pkg/wfs200/crs.go

```diff
@@ -0,0 +1,44 @@
+package wfs200
```

Comment:
Hier ben ik ook niet super enthousiast over. Ik had hier liever go interfaces gebruikt, maar dat lijkt niet de manier waarop deze library is opgebouwd.

Reply:
hoe bedoel je? snap ik niet:

```
Ik had hier liever go interfaces gebruikt
```

Reply:
Die crs-en zouden zich over de verschillende specifications ongeveer hetzelfde moeten gedragen. Behalve dat de wfs een uri moet zijn en de wms (liever) niet. Mijn idee zou zijn alleen dat gedrag te programmeren en de rest in een interface te vangen zodat het gedrag vastligt.

Reply:
Aah ja, het uitgangspunt is nu de xml structuur zoveel mogelijk benaderen in yaml. En aangezien WMS/WFS/WCS niet constistent zijn, is de yaml config niet consistent.